src/dialogs/stat_dialogs/survival/survival_model.pas

```diff
@@ -6,30 +6,37 @@ interface
 
 uses
   Classes, SysUtils, executor, epidatafiles, stat_dialog_contribution,
-  epidatafilestypes,epivaluelabels, outputcreator, freq, epicustombase;
+  epidatafilestypes,epivaluelabels, options_utils, epicustombase;
 
 type
 
-  TSurvivalStatDialogVariable = (tvX, tvY, tvW, tvBy);
+  TSurvivalStatDialogVariable = (tvX, tvT, tvT2, tvBy, tvW);
 
   { TSurvivalStatDialogVariableModel }
 
   TSurvivalStatDialogVariableModel = class(IStatDialogModel)
   private
     FExecutor: TExecutor;
     FXVariable: TEpiField;    // Outcome
-    FYVariable: TEpiField;    // Time
+    FTVariable: TEpiField;    // Time
```

Comment:
Yes - makes sense here.

Reply:
In my opinion we should name the varialble more closely to what they are used for - here I would choose FTimeVariable. It also eliminates the need for the comment since the code becomes self explanatory.
The problem with comments is they become obsolete over time and in the end may confuse because code has changed and the comment no longer represent the state of the code.

Reply:
And - yes - I know I haven't been consistent myself. It is just good practice to do so. :)

Reply:
I'm confused. This was changed.